base/pkg/entry.jl

```diff
@@ -376,7 +376,11 @@ function update(branch::AbstractString)
                         LibGit2.fetch(repo)
                         LibGit2.merge!(repo, fastforward=true)
                     catch err
-                        show(err)
+                        if isa(err, LibGit2.GitError)
+                            print("Package $pkg cannot be updated. Error: $(err.msg) Class: $(err.class)")
+                        else
+                            show(err)
```

Comment:
Swallowing errors leads to mystery situations like https://github.com/JuliaLang/julia/issues/13991 - if an exception happened, let's actually have it reported since something is probably broken. Allow through specific things we know are okay, not everything otherwise you end up having hidden typos again.